pandas/tests/computation/test_eval.py

```diff
@@ -1903,3 +1903,15 @@ def test_validate_bool_args(self):
         for value in invalid_values:
             with pytest.raises(ValueError):
                 pd.eval("2+2", inplace=value)
+
+
+def test_query_str_contains():
```

Comment:
add the issue number to this test. 

also name the test something more constructive. maybe test_with_str_accessor

see if this test can fit elsewhere in this file in a more logical section.

Reply:
Merged and updated tests